autoload/compe_tags/source.vim

```diff
@@ -26,6 +27,21 @@ function! s:datermine(context) abort
   return compe#helper#datermine(a:context)
 endfunction
 
+"
+" s:documentation
+"
+function! s:documentation(args) abort
+  let word = get(a:args.completed_item, 'word', '')
+  if empty(word)
+    return
+  endif
+  let tags = map(taglist(word), 'v:val.filename')
+  if len(tags) > 10
+    let tags = tags[0:9] + [printf('...and %d more', len(tags[10:]))]
+  endif
+  return a:args.callback(tags)
+endfunction
```

Comment:
Thank you! 
Could you add `l:` predix to local variable?
Sorry. I didn't setup CI yet.

And pleaae call `l:args.abort` instead with `return`.

Reply:
Fixed.